src/lib/ecore_x/ecore_x_api.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _EFL_ECORE_X_API_H
+#define _EFL_ECORE_X_API_H
+
+#ifdef ECORE_X_API
+#error ECORE_X_API should not be already defined
+#endif
+
+#ifdef _WIN32
+# ifndef ECORE_X_STATIC
+#  ifdef ECORE_X_BUILD
+#   define ECORE_X_API __declspec(dllexport)
+#  else
+#   define ECORE_X_API __declspec(dllimport)
+#  endif
+# else
+#  define ECORE_X_API
+# endif
+# define ECORE_X_API_WEAK
```

Comment:
@felipealmeida do we need a `weak` at `ecore_x`? This seems to be only used here at this file.

Reply:
And there's no `WAPI` on the original (master's) code.